### PR TITLE
server: add metrics to identify leader.

### PR DESCRIPTION
**Pull request by disksing**

### CODE REVIEW

File: server/metrics.go [hunk 1]
```diff
@@ -88,6 +88,14 @@ var (
 			Name:      "tso",
 			Help:      "Counter of tso events",
 		}, []string{"type"})
+
+	leaderTickCounter = prometheus.NewCounter(
```
Comment (siddontang) [inline server/metrics.go]: seem we can use tsoCounter to identify the leader.

Comment (disksing) [reply server/metrics.go]: tsoCounter does not work for rawkv.

Comment (siddontang) [reply server/metrics.go]: For every leader tick, we will update the TSO no matter we use txn KV or raw KV.
 https://github.com/pingcap/pd/pull/793/files#diff-2c671b5ee7007449d6115af14481769eR263

Comment (disksing) [reply server/metrics.go]: You are right :)

**Comment by siddontang** [comment]
I think we should add tso metric on Grafana @disksing

**Pull request closed without merging**